pkg/cli/build.go

```diff
@@ -272,8 +271,8 @@ func GenBuildOptions(c *cobra.Command, inputArgs []string, iopts BuildOptions) (
 		return options, nil, nil, err
 	}
 
-	decryptConfig, err := iutil.DecryptConfig(iopts.DecryptionKeys)
-	if err != nil {
+	decryptConfig, err := util.DecryptConfig(iopts.DecryptionKeys)
+	if err != nil || decryptConfig == nil {
```

Comment:
`decryptConfig == nil` means “don’t decrypt”, and it’s what I propose we do by default; failing on that doesn’t make sense to me.